draft-ietf-acme-acme.md

```diff
@@ -2572,6 +2574,14 @@ checks, then the validation fails.
 
 ## MIME Type: application/pem-certificate-chain
 
+A file of this type contains one or more certificates encoded with the PEM textual encoding, according to
+RFC 7468 {{!RFC7468}}.  In order to provide easy interoperation with TLS, the first
+certificate MUST be an end-entity certificate. Each following certificate
+SHOULD directly certify the one preceding it. Because certificate validation
+requires that trust anchors be distributed independently, a certificate
+that specifies a trust anchor MAY be omitted from the chain, provided
```

Comment:
This says "a certificate that specifies a trust anchor MAY be omitted from the chain" but that makes it sound like the certificate whose issuer is a trust anchor specified out of band can be omitted when I think the intention is that the trust anchor itself may be omitted.

Can you clarify this? Maybe:
> Because certificate validation requires that trust anchors be distributed independently required trust anchors MAY be omitted from the chain, provided that supported peers are known to possess the omitted trust anchor certificates

(or similar?)

Reply:
I didn't change the text here, just copy/pasted it from below :)  Nonetheless, happy to use your text.

Reply:
I'm just going to change "specifies" to "represents".  I think that addresses the ambiguity you note; it doesn't seem like a cert under a TA would be understood to represent that TA.

Reply:
> I didn't change the text here, just copy/pasted it from below

:laughing: Doh!

> I'm just going to change "specifies" to "represents". I think that addresses the ambiguity you note; it doesn't seem like a cert under a TA would be understood to represent that TA.

That works for me :+1: Thanks